src/files.h

```diff
@@ -140,7 +140,8 @@ R"RAW(
 #endif
 R"RAW(
         "close_tab": "<primary>w",
-        "window_toggle_split": ""
+        "window_toggle_split": "",
+        "clear_console": "<primary><shift>c"
```

Comment:
Please use window_clear_console. The default key could be "" in case we want to use control-shift-c for something else later. Or?